ksql-engine/src/main/java/io/confluent/ksql/ddl/commands/CreateStreamCommand.java

```diff
@@ -52,9 +52,11 @@ public DDLCommandResult run(MetaStore metaStore) {
         sourceName,
         schema,
         (keyColumnName.length() == 0)
-          ? null : SchemaUtil.getFieldByName(schema, keyColumnName).orElse(null),
+        ? null
```

Comment:
I think the changes in this and `CreateTableCommand` should be reverted. I presume you added them to fix the checktyle failures, but they have been fixed upstream already.